build/data_emoji_names_v5.txt

```diff
@@ -1,14 +1,14 @@
 1F6F7;sled
 1F6F8;flying_saucer
 1F91F;i_love_you_hand_sign
-1F928;face_with_one_eyebrow_raised
-1F929;grinning_face_with_star_eyes
-1F92A;grinning_face_with_one_large_and_one_small_eye
-1F92B;face_with_finger_covering_closed_lips
-1F92C;serious_face_with_symbols_covering_mouth
-1F92D;smiling_face_with_smiling_eyes_and_hand_covering_mouth
-1F92E;face_with_open_mouth_vomiting
-1F92F;shocked_face_with_exploding_head
+1F928;face_with_raised_eyebrow/face_with_one_eyebrow_raised
+1F929;star-struck/grinning_face_with_star_eyes
```

Comment:
this should be `_`

Reply:
the CLDR name is "star-struck" with the hyphen